FinnHub.Core/FinnHub.Core.csproj

```diff
@@ -23,8 +23,11 @@
   </PropertyGroup>
 
```

Comment:
Make sure to bump the version to 1.0.1 and make sure that is both the NuGet Package Version and the .NET Assembly Version.